src/schema/v2/artwork/__tests__/artwork.test.js

```diff
@@ -17,7 +17,7 @@ describe("Artwork type", () => {
     {
       artistId: "artist-id",
       demandRank: 0.64,
-      medium: "Print",
+      medium: "print",
```

Comment:
this was breaking the comparison here https://github.com/artsy/metaphysics/blob/0ea0fbc063d1c29a5ea38f68cc30d4800af2b19a/src/lib/fillers/enrichArtworksWithPriceInsights.ts-L32